client/src/components/ReviewTile.js

```diff
@@ -1,15 +1,59 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import ReviewVotes from './ReviewVotes'
 
 const ReviewTile = (props) => {
-  
   const shortDate = (props.review.createdAt).slice(0, 10)
 
+  const deleteReview = async () => {
+    try {
+      const response = await fetch(`/api/v1/sandwiches/${props.review.sandwichId}/reviews`, {
```

Comment:
not sure if this was in my initial review. If not, pardon for volleying this back at you, but this path is too nested.

Instead, it should be `/api/v1/reviews/:id` and the method should be DELETE. 

The reason is that we dont need the sandwich in order to delete a review, so there is no reason to have the sandwich in the URL. This means it needs to go to a `reviewsRouter.delete(/:id...)` endpoint instead. If you dont have a reviewsRouter, now is the time to make one.

your edit functionality will be similar, in that you wont need a sandwich to make an edit for an existing review